target_core_user_local.h

```diff
@@ -69,6 +69,7 @@ struct tcmu_cmd_entry_hdr {
 	__u16 cmd_id;
 	__u8 kflags;
 #define TCMU_UFLAG_UNKNOWN_OP 0x1
+#define TCMU_UFLAG_COMPLETED  0x2
```

Comment:
@mikechristie 

This should be updated in kernel too. I think should we release one new version for target_core_user.ko for this.